docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_independent_evaluation.yml

```diff
@@ -259,17 +255,6 @@ question: |
   Are you comfortable sharing your income information with ${ school.recipient.name }?
 yesno: comfortable_sharing_income
 ---
-sets:
```

Comment:
Why is this question being entirely removed?

Reply:
This id is already in the basic